README.md

```diff
@@ -111,3 +111,100 @@ IEEEDataPort,
+| Huch et al.        |       2023 | Quantifying the LiDAR Sim-to-Real Domain Shift: A Detailed Investigation Using Object Detectors and Analyzing Point Clouds at Target-Level | [10.1109/TIV.2023.3251650](doi.org/10.1109/TIV.2023.3251650) | [Sim2Real-DistributionAligned-Dataset](https://github.com/TUMFTM/Sim2RealDistributionAlignedDataset) | Lidar | Technical University of Munich  | / |
+| Haider et al.      |       2023 | Performance Evaluation of MEMS-Based Automotive LiDAR Sensor and Its Simulation Model as per ASTM E3125-17 Standard | [10.3390/s23063113](https://doi.org/10.3390/s23063113) | / |Radar | Kempten University of Applied Sciences | VIVID |
+| Ngo                |       2023 | A methodology for validation of a radar simulation for virtual testing of autonomous driving | [10.18419/opus-12703](http://dx.doi.org/10.18419/opus-12703) | / | Radar | University of Stuttgart  | Robert Bosch GmbH |
+| Rosenberger        |       2023 | Metrics for Specification, Validation, and Uncertainty Prediction for Credibility in Simulation of Active Perception Sensor Systems | [10.26083/tuprints-00023034](https://doi.org/10.26083/tuprints-00023034) | / | Generic (Lidar) | TU Darmstadt   | ENABLE-S3, PEGASUS, SET Level, VVMethods |
+| Ngo et al.         |       2021 | A Multi-Layered Approach for Measuring the Simulation-to-Reality Gap of Radar Perception for Autonomous Driving | [10.1109/ITSC48978.2021.9564521](https://doi.org/10.1109/ITSC48978.2021.9564521) | / | Radar | University of Stuttgart  | Robert Bosch GmbH |
+| Schaermann         |       2020 | Systematische Bedatung und Bewertung umfelderfassender Sensormodelle | [mediaTUM](https://mediatum.ub.tum.de/doc/1518611/1518611.pdf) | / | Generic (Lidar) | TU München  | BMW AG |
+| Rosenberger et al. |       2019 | Towards a Generally Accepted Validation Methodology for Sensor Models - Challenges, Metrics, and First Results | [TUprints](http://tubiblio.ulb.tu-darmstadt.de/113867/) | / | Generic    | TU Darmstadt   | PEGASUS    |
+
+##### Measurement Campaigns for Sensor Model Validation
+
+| Authors            | Date       | Title                            | Chapter(s) | Link / Repo / Paper / DOI                            | Modality?  | Facility  | Funding |
+| ------------------ | ---------- | -------------------------------- | ---------------------------------------------------- | ---------- | ---------- | --------- | ------- |
+| Linnhoff           |       2023 | Analysis of Environmental Influences for Simulation of Active Perception Sensors | Chapter 5 | [10.26083/tuprints-00023116](https://doi.org/10.26083/tuprints-00023116) | Lidar | TU Darmstadt   | SET Level, VVMethods |
+| Holder             |       2021 | Synthetic Generation of Radar Sensor Data for Virtual Validation of Autonomous Driving | Chapters 4.2, 5.2, 6.2-6.4 | [10.26083/tuprints-00017545](https://doi.org/10.26083/tuprints-00017545) | Radar | TU Darmstadt   | SET Level, VVMethods |
+
```

Comment:
Should we add here:
https://ieeexplore.ieee.org/document/10025706
and
https://assets.researchsquare.com/files/rs-3088648/v1_covered_2fc41b7c-2dbc-4ad9-b58e-f99d21c33e9d.pdf?c=1687488148
?

Reply:
Yes, looks good. Can you add them?

Reply:
Done.